apertium-uzb.uzb.lexc

```diff
@@ -1,64 +1,106 @@
+%<abe%>       ! Abessive=Privative         ! Лишительный
+              ! # SIZ (not used after posessives and cases)
+%<reas%>      ! not used rigth now, just in case for # LIKDAN
+
+! Levels of comparison of adjectives   ! Sifatlarning solishtirma darajalari
+%<comp%>      ! Comparative            ! Solishtirma
+
+! TODO
+!! Some adjecives have a "weakening" form, e.g. oq>oqish, sariq>sarg'ish; but
+!! also oq>oqimtir, qora>qoramtir. Sometimes they take the comparative ROQ affix,
+!! e.g. qizg'ishroq.
+
+! TODO
+!! There is also what can be called a synthetic superlative form (as opposed
+!! to the /eng<adv>+X<adj>/ construction) -- qop-qora, sap-sariq, ko'm-ko'k etc.
+!! Probably easiest way would be just to lexicalize them, but make sure that
+!! they don't take the comparative affix.
+
 
 ! Possessives
-%<px1sg%>     ! First person singular
-%<px2sg%>     ! Second person singular
-%<px3sp%>     ! Third person singular/plural
-%<px1pl%>     ! First person plural
-%<px2pl%>     ! Second person plural
-%<px3pl%>     ! Third person plural (for reflexive)
-!%<px%>        ! General possessive          !# -нікі  ! now gen.subst
+%<px1sg%>     ! First person singular                   ! 1-shaxs birlik
+%<px2sg%>     ! Second person singular                  ! 2-shaxs birlik
+%<px3sp%>     ! Third person singular/plural            ! 3-shaxs birlik/ko'plik
+%<px1pl%>     ! First person plural                     ! 1-shaxs ko'plik
+%<px2pl%>     ! Second person plural                    ! 2-shaxs ko'plik
+%<px3pl%>     ! Third person plural (for reflexive)     ! 3-shaxs ko'plik
+!%<px%>        ! General possessive                     ! # -niki  ! now gen.subst
 
-!!!!!! MISC!!!!
 
-%<top%> ! TOPONYMS
-%<al%> ! ALTER
-%<ant%>       ! Anthroponym
-%<cog%>       ! Cognomen     ! Фамилия
-%<org%>
-%<al%> !others
+! Proper noun types          ! Atoqli otlar turari
+%<top%>       ! Toponym      ! Toponim
+%<ant%>       ! Anthroponym  ! Shaxs ismi
+%<cog%>       ! Cognomen     ! Familiya
+%<pat%>       ! Patronym     ! Sharif/ Otchestva
+%<org%>       ! Organisation ! Korxona-Tashkilot nomi
+%<al%>        ! Other        ! Boshqalar
+
+!! Gender of anthroponyms and cognoms  !! Shaxs ismi va familiyasi jinsi
+%<m%>         ! Masculine          ! Erkak
+%<f%>         ! Feminine           ! Ayol
+%<mf%>        ! Masculine/feminine !# basically cognoms without -ov/-ova,-in/-ina endings
+
+! Verbal features   ! Fel xususiyatlari
+
+!! Mood                          ! Kayfiyatni bildiruvchi
+%<imp%>       ! Imperative       ! Imperativ
+%<opt%>       ! Optative/jussive ! ?? / Imported from Kazakh, not used yet
+%<evid%>      ! Evidential, a.k.a. "indirect" / non-eyewitness / hearsay
 
-%<m%> ! 
-%<f%> !
-%<mf%> !
+!! Derivation                ! Kelib chiqishiga qarab
+%<caus%>      ! Causative    ! ?? / Imported from Kazakh, not used yet
+%<pass%>      ! Passive      ! Passiv
+%<coop%>      ! Cooperative  ! ?? / Imported from Kazakh, not used yet
```

Comment:
These all exist in Uzbek; we should discuss them and their implementation.

Reply:
Noted.